### PR TITLE
DAT-16095 DevOps :: Configure liquibase-teredata repo to use build-logic

### PR DESCRIPTION
**Pull request by jandroav**
https://datical.atlassian.net/browse/DAT-16095

### CODE REVIEW

File: .github/workflows/release-published.yml [hunk 1]
```diff
@@ -1,80 +1,52 @@
...
-          echo "Publishing version ${{ steps.collect-data.outputs.version }} from ${{ steps.collect-data.outputs.tag }}"
+  release:
+    uses: liquibase/build-logic/.github/workflows/extension-release-published.yml@v0.5.2
+    secrets: inherit
 
-  deploy_maven:
-    name: Deploy to Maven
-    needs: [ setup ]
-    runs-on: ubuntu-latest
+  deploy_xsd:
```
Comment (jnewton03) [inline .github/workflows/release-published.yml:13]: @jandroav same thing here.  Is this needed?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      - opened
+      - reopened
+      - synchronize
+
+jobs:
+  build:
+    uses: liquibase/build-logic/.github/workflows/os-extension-test.yml@v0.5.2
+    secrets: inherit
+
+#  integration-test:
```
Comment (jnewton03) [inline .github/workflows/test.yml:16]: @jandroav do we want to run integration tests?  Wouldn't we do this just like neo4j where the build/unit tests are reusable and then the integration tests are a separate step?

Comment (jandroav) [reply .github/workflows/test.yml:16]: Yes. Same as maxdb. Tests were commented on in the original workflow (ci.yml). We can uncomment them and see what happens. Who should take a look if tests have failures?

File: .github/workflows/release-published.yml (hunk 1 above)
Comment (jandroav) [reply .github/workflows/release-published.yml:13]: Nop, sorry. Bad copy and paste

File: .github/workflows/test.yml (hunk 2 above)
Comment (jnewton03) [reply .github/workflows/test.yml:16]: @jandroav ah ok.  Lets handle that in separate ticket.  @filipelautert @kevin-atx who is the maintainer of this extension that could re-enable those tests?

File: .github/workflows/release-published.yml (hunk 1 above)
Comment (jandroav) [reply .github/workflows/release-published.yml]: fixed

File: .github/workflows/test.yml (hunk 2 above)
Comment (jandroav) [reply .github/workflows/test.yml]: yes because no tests are executed:

```
      matrix:
       teradata: [ "" ]
```

**Pull request merged**